### DISCUSSION TITLE
Filter in Tables

### DISCUSSION THREAD

**Discussion by RadheshRagunath** [squidfunk/mkdocs-material#2584] [Q&A] [answered]
Is it possible to include a filter option to the tables designed with mkdocs?

Thank you for your suggestions!!

**Comment by squidfunk** [comment, marked as answer]
This could surely be done with some additional JavaScript, see for example [this StackOverflow post](https://stackoverflow.com/questions/51187477/how-to-filter-a-html-table-using-simple-javascript) which contains executable examples.

**Reply by RadheshRagunath** [reply]
Thank you very much @squidfunk 
Will look into the above link!!
